.github/workflows/camera_cq.yaml

```diff
@@ -12,8 +12,22 @@ jobs:
     - name: wemake-python-styleguide
       uses: wemake-services/wemake-python-styleguide@0.13.4
       with:
-        path: 'camera/'
+        path: 'camera/rpi-files/'
         reporter: 'github-pr-review'
       env:
         GITHUB_TOKEN: ${{ secrets.github_token }}
```

Comment:
is it your account token? If yes then think of marking it somehow to give aus the ability to use ours also in CI

Reply:
This token is something autogenerated per repository each 60 minutes: https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#about-the-github_token-secret